src/vm/win32threadpool.cpp

```diff
@@ -1815,8 +1815,8 @@ Thread* ThreadpoolMgr::CreateUnimpersonatedThread(LPTHREAD_START_ROUTINE lpStart
         // CreateNewThread takes care of reverting any impersonation - so dont do anything here.
         bOK = pThread->CreateNewThread(0,               // default stack size
                                        lpStartAddress,
-                                       lpArgs           //arguments
-                                       );
+                                       lpArgs,           //arguments
+                                       W("ThreadPool"));
```

Comment:
So far the PR should be complete. Background GC threads show up properly. The only thing I have not found are where server GC threads are created but that is not a major issue since not many people are using server GC except inside IIS.